src/app/components/Icons/SearchIcon.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+function SearchIcon(): JSX.Element {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="22.576"
+      height="23.826"
+      viewBox="0 0 22.576 23.826"
+    >
+      <g id="Group_89" data-name="Group 89" transform="translate(963 -79)">
```

Comment:
```suggestion
      <g transform="translate(963 -79)">
```
Is this really needed?